harvard-university-of-leeds.csl

```diff
@@ -153,7 +157,7 @@
             <text term="in" text-case="capitalize-first" font-style="italic" suffix=":"/>
             <names variable="editor" delimiter=", " prefix=" " suffix=".">
               <name and="text" delimiter-precedes-last="never" initialize-with=". ">
-                <name-part name="family" text-case="uppercase"/>
+                <name-part name="family" text-case="capitalize-first"/>
```

Comment:
yeah -- most of the time this does nothing; in some very rare cases it probably does things you don't want (like bell Hooks or e.e. Cummings)